source/Calamari.Shared/Integration/Scripting/Python/InstallDependencies.py

```diff
@@ -0,0 +1,19 @@
+import importlib
+import subprocess
+import sys
+
+def install_missing_dependency(module, package):
+    try:
+        printverbose("Checking for dependency {}".format(package))
+        importlib.import_module(module)
+        printverbose("{} was found".format(package))
+    except:
+        printverbose("Did not find dependency, attempting to install {} for the current user using pip".format(package))
+        subprocess.call([sys.executable, "-m", "pip", "install", package, "--user"])
```

Comment:
pip isn't always installed by default with python... Not sure if you're handling this case.

Reply:
Suggested we provide a helpful message with some steps the user can take (similar to what we do for IIS) to ensure that they can run python scripts on their targets (i.e. install `pip` or install the `pycryptodome` package using their package manager of choice on the target)